translation/source/site.xml

```diff
@@ -756,7 +714,6 @@ computer analysis, game chat and shareable URL.</string>
   <string name="error.maxLength">Maximum length is %s</string>
   <string name="error.min">Must be greater or equal to %s</string>
   <string name="error.max">Must be less or equal to %s</string>
-  <string name="error.unknown">Error</string>
```

Comment:
Is this maybe automatically used by play? Otherwise, `error.min` and `error.max` can also be removed.

Reply:
will investigate